src/panel/App.css

```diff
@@ -12,3 +12,13 @@ code,
 pre {
   font-family: "Roboto Mono", monospace;
 }
+
+button {
```

Comment:
Yeah, good point. My preference is always to reset on a global level to avoid bloating compnoent styles but I can see that the current approach is to unset on a per-component basis. If we went with a global reset for all HTML elements there'd be a good cleanup opportunity?

@andyrichardson any thoughts on this?